### PR TITLE
feat: add doReload action

### PR DESCRIPTION
**Pull request by Caele**
Adds doReload as an Action, with the partial flag possible.
My prettier settings did some things, but the lint passed so..

### CODE REVIEW

**Review by niekvanstaveren — APPROVED** [review]
Looks good. I think lint has been updated quite a bit since we did this work 😄

File: src/utils/actions.js [hunk 1]
```diff
@@ -218,10 +218,22 @@ const actions = [
     },
     requiredInput: ['variable', 'value'],
   },
+  {
+    value: 'doReload',
+    translation: 'Object.ActionButton.DoReload',
+    group: 'reload',
+    getActionCall: ({ app, partial }) => async () => {
+      const e = await app.doReload(0, !!partial, false);
+      if (e) {
```
Comment (haxxmaxx) [inline src/utils/actions.js:227]: What is e? error?

**Comment by haxxmaxx** [comment]
Some unit tests would be nice :)

File: src/utils/actions.js (hunk 1 above)
Comment (Caele) [reply src/utils/actions.js:227]: true or false, if the reload succeded or not.
I'm not checking for any permissions etc, so if you try to do a reload without update permissions I assume you'll get an generic "Access denied" message as well.

**Pull request merged**